### DISCUSSION TITLE
How long do updates usually take?

### DISCUSSION THREAD

**Discussion by Vistalgia** [m4dEngi/RemotePlayWhatever#121] [Q&A]
Title.

I've been enjoying Baldur's Gate 3 via split-screen coop with a friend online with the use of this app. Lately, Steam was updated and made it impossible to launch :(

**Comment by m4dEngi** [comment]
Somewhere around 1-2 days, updates can't be fully automated and I don't always have time to update immediately when a new client comes out.
